app/views/devise/sessions/new.html.erb

```diff
@@ -1,11 +1,4 @@
 <% title t('titles.visitors.index') %>
-<% if @browser_is_ie11 %>
```

Comment:
Should we keep this but update the language so that we tell IE11 users that the browser is not supported anymore. If we remove this IE11 users wont get an indication that the website wont work as intended for them.

Reply:
> Should we keep this but update the language so that we tell IE11 users that the browser is not supported anymore. If we remove this IE11 users wont get an indication that the website wont work as intended for them.

That's a good question! I'm not sure. I think you make a good point that it would be nice for the user to be informed that their browser is not supported. On the other hand, how long would we want to maintain that? Forever? Also, there is a small subset of browsers already which we technically do not support but which have not and would not see any messaging about browser support (typically older versions of Chrome, etc).

This also reminds me of a [past conversation](https://gsa-tts.slack.com/archives/C0NGESUN5/p1661799608804059) about how a good chunk of the frontend errors we're logging are coming from said unsupported browsers, and how we might want to hide or ignore those. At the time I'd contemplated whether we could have code which detects whether the browser is supported and suppresses error logging. I could see how an approach like that (independent of IE) could also be used as a hook for displaying some sort of message about the current browser being unsupported.

Reply:
I am in favor of showing a banner to users that their browser is not supported

Reply:
oo yes that sounds like a better idea if we want to go about this route. I think a separate ticket to possibly implement this would reduce some of the weirdness we may see with users.